static/js/snap-pages/components/PublisherMetrics/PublisherMetrics.tsx

```diff
@@ -0,0 +1,54 @@
+import { Col, Notification, Row, Strip } from "@canonical/react-components";
+import { ISnap } from "../../types";
+import { renderPublisherMetrics } from "../../../publisher-pages/pages/Metrics/metrics/metrics";
+import { useEffect } from "react";
+import { useFetchPublishedSnapMetrics } from "../../hooks";
+
+export const PublisherMetrics = ({ snaps }: { snaps: ISnap[] }) => {
```

Comment:
I think this should be a named function and the export at the bottom of the file to match other React components